### DISCUSSION TITLE
Is there the same "CTRL - Scale" function as Bambu Studio or PrusaSlicer?

### DISCUSSION THREAD

**Discussion by dasFelix** [OrcaSlicer/OrcaSlicer#3662] [Ideas]
Assumption: You want scale an object, uniform scaling is already off.

When you have an object and press (S) Scale you can click the X (green) or Y (red) anchor points and scale the object in the desired dimension.
But scaling always works from the center in both directions. It's the same with PrusaSlicer (2.7.1) or Bambu Studio (1.8.3.89). BUT here you can select the anchor point with CTRL-click, then the object actually scales only in this selected direction.

I miss this in OrcaSlicer (1.9.0). Or is it a different key combination and/or setting which I haven't found yet?

This function is especially helpful when you want to align two objects to each other. Imagine as simple example a frame that you want to widen with some cube primitives.

**Comment by LJKraus** [comment]
I think there might be a bug here.  In OrcaSlicer 1.9, click and drag the anchor, and both sides scale equally.  With CTRL-click, the object expands primarily in the intended direction, but the opposite side also moves.  It's not much, but it does not stay in position the way it does in Studio.

**Comment by chuckkay** [comment]
Used to work in orca 18, totally sucks that its broke and does a useless move instead of one sided scale. I Find myself switching back to bambu when i need to modify models..  another complaint is with the cut planar tool, it shows colors magenta and cyan on the object but not corresponding colors in the dialog box  like bambu does making it even harder to determine considering orca has also shrunk the cut rotation controls so small..… I do love the new text tools, im totally fine with them not being compatible for editing with older 3mf’s created before 19, but unless Bambu Adopts orcas new approaches to text, any text created in Bambu Studio going forward will not be editable in orca and visa versa, i would hope Bambu would adopt orca’s approach to text but I doubt it, because they may find the loss of editing backwards compatible 3mfs upsetting to customers
